manifest/properties.yaml

```diff
@@ -1,68 +1,68 @@
 convergence_ionic:
   defaults:
     units: eV
-  schemaId: properties-directory-workflow-convergence-ionic
+  schemaId: properties-directory/workflow/convergence/ionic 
```

Comment:
Wow - how did this even work?

Reply:
it didn't 😄 all `express` unit tests were broken when I updated the latest version of esse. But yeah, we should probably write a test for that, or perhaps there is another way that doesn't require mapping the property slug to the `schemaId`.